### DISCUSSION TITLE
2 way data binding to selected table rows

### DISCUSSION THREAD

**Discussion by corey-Robinson1337** [Power-Components/livewire-powergrid#1257] [Q&A]
I would like to set which checkboxes are selected in the table when it's loaded.

Can anybody provide instruction on how to do this? Ideally, I would like to just provide an array of the id's that are selected and bind that to what the user selects. Any help with this would be great!!

Thanks.

**Comment by abhaypithadiya** [comment]
Hey! 

The $this->checkboxValues variable holds the selected ids, you can add the ids in this array and it will be selected in the table when it loads.

I am not sure if this is what you need. Let me know if it works out.

```
    public function setUp(): array
    {
        $this->showCheckBox();

        $this->checkboxValues = [1,2,3];   <---- This line here

        return [
            Exportable::make(Carbon::today()->format('d-m-Y').'-users')
                ->striped()
                ->type(Exportable::TYPE_XLS, Exportable::TYPE_CSV),
            Header::make()->showSearchInput(),
            Footer::make()
                ->showPerPage(10, [10, 25, 50, 100])
                ->showRecordCount(),
        ];
    }
```

**Reply by corey-Robinson1337** [reply]
hey @abhaypithadiya that works and is pretty similar to what I ended up doing.
